### PR TITLE
Drawables changed for common error icon

### PR DESCRIPTION
**Pull request by sd1998**
Fie to issue #2473.

### CODE REVIEW

File: owncloudApp/src/main/res/layout/share_public_dialog.xml [hunk 1]
```diff
@@ -190,15 +190,15 @@
...
                     android:drawablePadding="5dp"
-                    android:drawableStart="@drawable/common_error"
+                    android:drawableStart="@drawable/common_error_black"
                     android:gravity="center_vertical"
                     android:paddingLeft="@dimen/standard_half_padding"
                     android:paddingRight="@dimen/standard_half_padding"
                     android:textColor="@color/secondaryTextColor"
                     android:textSize="13sp"
-                    android:visibility="gone" />
+                    android:visibility="visible" />
```
Comment (abelgardep) [inline owncloudApp/src/main/res/layout/share_public_dialog.xml]: Error should be gone until error triggers

Comment (sd1998) [reply owncloudApp/src/main/res/layout/share_public_dialog.xml]: @abelgardep Sorry, just an error while testing.

**Comment by abelgardep** [comment]
In which status is this, @sd1998?

**Comment by sd1998** [comment]
@jesmrec @abelgardep I have made the changes.

**Comment by abelgardep** [comment]
Thanks. Looks good to me, what do you think @davigonz ?

**Comment by abelgardep** [comment]
Ready to test @jesmrec

**Comment by davigonz** [comment]
> Thanks. Looks good to me, what do you think @davigonz ?

Go ahead!!

tenor

**Comment by jesmrec** [comment]
Approved

**Comment by sd1998** [comment]
@jesmrec I have done the rebase.

**Comment by jesmrec** [comment]
Thanks a lot!! merged and ready for 2.11

**Pull request merged**